packages/react-sdk/docusaurus/docs/React/02-guides/10-sorting-api.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Participant sorting
+description: Overview of the Sorting API used to sort call participants.
+---
+
+## Introduction
+
+The Participant Sorting API is a powerful tool built on top of the internal `Comparator<T>` API,
+providing developers with the ability to sort participants in various scenarios. This API offers common comparators and built-in presets that can be easily customized or used out-of-the-box, making participant sorting a seamless experience.
+
+When dealing with real-time communication applications, it is often necessary to sort participants based on specific criteria.
+Whether you need to adjust the sorting in existing view layouts or define new sorting presets, the **Participant Sorting API** is here to simplify the process.
+
+By utilizing the `Comparator<T>` API and the provided built-in comparators and presets, developers can effortlessly sort participants according to their requirements.
+
+## `Comparator<T>` API overview
+
+The `Comparator<T>` API is the foundation upon which the Participant Sorting API is built.
+It defines a function type `Comparator<T>` that takes two arguments `a` and `b` of type `T` and returns `-1`, `0`, or `1` based on the comparison between the two items.
```

Comment:
I'd mention that it's an extension of the Array.sort API so people know what the return values of 1, 0 and -1 mean.

Reply:
True. I'll add this information.

Reply:
Updated.